webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue

```diff
@@ -26,7 +26,10 @@
       {{ $t('activity.composer.title') }}
     </template>
     <template #content>
-      <v-card flat>
+      <v-card 
+        flat
+        class="full-height" 
+        v-draggable="draggableObjectParam">
```

Comment:
Move the definition of directive in drawer, by using `this.$refs.drawer.$el` to give access to drawer DOM element

```suggestion>
```

Reply:
done